From c977209c36ece39e79c4b14b8d933819fe4eeec6 Mon Sep 17 00:00:00 2001
From: Ferdinand Xu <cheng.a.xu@intel.com>
Date: Mon, 7 Sep 2015 03:06:41 -0400
Subject: [PATCH 0638/1164] CDH-40980: HIVE-11747: Unnecessary error log is
 shown when executing a "INSERT OVERWRITE LOCAL
 DIRECTORY" cmd in the embedded mode(Ferdinand Xu,
 reviewed by Dong Chen)

Change-Id: I7b5eb4daffc4dc261a722eaf8a884f10835d9549
---
 .../apache/hadoop/hive/ql/exec/mr/ExecDriver.java  |    1 -
 1 file changed, 1 deletion(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/exec/mr/ExecDriver.java b/ql/src/java/org/apache/hadoop/hive/ql/exec/mr/ExecDriver.java
index 72de9f9..85e3037 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/exec/mr/ExecDriver.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/exec/mr/ExecDriver.java
@@ -396,7 +396,6 @@ public int execute(DriverContext driverContext) {
       if (pwd != null) {
         HiveConf.setVar(job, HiveConf.ConfVars.METASTOREPWD, "HIVE");
       }
-      LOG.error(job.get("mapreduce.framework.name"));
       JobClient jc = new JobClient(job);
       // make this client wait if job tracker is not behaving well.
       Throttle.checkJobTracker(job, LOG);
-- 
1.7.9.5

